src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonNumberEnumConverter.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json.Serialization.Converters;
+
+namespace System.Text.Json.Serialization
```

Comment:
How did this class not end up in System.Text.Json.Serialization.Converters?

Reply:
I think that namespace is only used for internal converters, I don't think it's necessary for public stuff.